### PR TITLE
make publish test like way faster

### PR DESCRIPTION
**Pull request by mikehelmick**
reuse database for each test

### CODE REVIEW

File: internal/publish/publish_test.go [hunk 1]
```diff
@@ -129,13 +130,40 @@ func issueJWT(t *testing.T, cfg jwtConfig) (jwtText, hmacKey string) {
...
+	}
+}
+
+func (n *nameAssigner) next() string {
+	n.modifier++
+	n.lastAssigned = fmt.Sprintf("%s.%d", n.baseAPK, n.modifier)
+	return n.lastAssigned
+}
+
+func (n *nameAssigner) last() string {
```
Comment (whaught) [inline internal/publish/publish_test.go:151]: I might call it "peek()" or "current()" given that it's the only one kept around.

Comment (mikehelmick) [reply internal/publish/publish_test.go]: renamed to current

**Pull request merged**